packages/app/src/cli/prompts/deploy-release.ts

```diff
@@ -1,3 +1,4 @@
+import {buildNURInfoTableSection} from './ui/NURInfoTableSection.js'
```

Comment:
nit, what is `NUR`? "new", "updated", "removed"? Can we de-acronym this for better readability?